br/pkg/lightning/importer/checksum_helper.go

```diff
@@ -60,7 +60,13 @@ func NewChecksumManager(ctx context.Context, rc *Controller, store kv.Storage) (
 			log.FromContext(ctx).Info("set tidb_backoff_weight to default", zap.Int("backoff_weight", local.DefaultBackoffWeight))
 			backoffWeight = local.DefaultBackoffWeight
 		}
-		manager = local.NewTiKVChecksumManager(store.GetClient(), pdCli, uint(rc.cfg.TiDB.DistSQLScanConcurrency), backoffWeight)
+
+		explicitRequestSourceType, err := common.GetExplicitRequestSourceTypeFromDB(ctx, rc.db)
```

Comment:
Will it fail when the target cluster is lower version? Is this by design?

Reply:
- Querying a lower version of tidb only returns empty results.
- When getting empty results, it will be the same as the default setting.